nebula-algorithm/README.md

```diff
@@ -1,75 +1,85 @@
     $ cd nebula-algorithm
     $ mvn clean package -Dgpg.skip -Dmaven.javadoc.skip=true -Dmaven.test.skip=true
     ```
-    编译完成后，在 `nebula-algorithm/target` 目录下会生成 `nebula-algorithm-2.0.0.jar` 。
+    After the above buiding process the target file  `nebula-algorithm-2.0.0.jar` will be placed under `nebula-algorithm/target` .
 
- 2. 在 Maven 远程仓库下载
-   https://repo1.maven.org/maven2/com/vesoft/nebula-algorithm/2.0.0/
+ 2. Download from Maven repo
+      
+      Alternatively, it could be downlaoded from below Maven repo:
+      
+      https://repo1.maven.org/maven2/com/vesoft/nebula-algorithm/2.0.0/
 
-# 使用 Nebula-Algorithm
+## Use Nebula-Algorithm
 
-   使用限制：Nebula-Algorithm 未自动对字符串id进行编码，因此执行图算法时，边的源点和目标点必须是整数（Nebula Space 的 vid_type可以是String类型，但数据必须是整数）。
-   
-* 使用方法1：直接提交 nebula-algorithm 算法包
+Limitation: Due to Nebula-Algorithm will not encode string id, thus during the algorithm execution, the source and target of edges must be in Type Int (The `vid_type` in Nebula Space could be String, while data must be in Type Int).
+
+* Option 1: Submit nebula-algorithm package
 
-   * 设置配置文件
+   * Configuration
    
-    关于配置项的具体说明参考[示例配置](https://github.com/vesoft-inc/nebula-spark-utils/blob/master/nebula-algorithm/src/main/resources/application.conf)
+   Refer to [configuration example](https://github.com/vesoft-inc/nebula-spark-utils/blob/master/nebula-algorithm/src/main/resources/application.conf)
 
-   * 提交算法任务
+   * Submit Spark Applicaiton
 
     ```
     ${SPARK_HOME}/bin/spark-submit --master <mode> --class com.vesoft.nebula.algorithm.Main nebula-algorithm-2.0.0.jar -p application.conf
     ```
-* 使用方法2：调用 nebula-algorithm 算法接口
+   
+* Option2: Call nebula-algorithm interface
 
-   在`nebula-algorithm`的`lib`库中提供了10中常用图计算算法，可通过编程调用的形式调用算法。
-   * 在pom.xml中添加依赖
+   Now there are 10 algorithms provided in `lib` from `nebula-algorithm`, which could be invoked in a programming fashion as below:
+   
+   * Add depedencies from `pom.xml`
    ```
     <dependency>
          <groupId>com.vesoft</groupId>
          <artifactId>nebula-algorithm</artifactId>
          <version>2.0.0</version>
     </dependency>
    ```
-   * 定义算法参数调用算法（以`PageRank`为例）
+   * Pass arguments of the algorithm, below is an example for `PageRank`.
    ```
    val prConfig = new PRConfig(5, 1.0)
    val louvainResult = PageRankAlgo.apply(spark, data, prConfig, false)
    ```
- 
-    其他算法的调用方法见[测试示例](https://github.com/vesoft-inc/nebula-spark-utils/tree/master/nebula-algorithm/src/test/scala/com/vesoft/nebula/algorithm/lib) 。
-    
-    *注：执行算法的DataFrame默认第一列是源点，第二列是目标点，第三列是边权重。*
+   
+    For other algorithms, please refer to [test cases](https://github.com/vesoft-inc/nebula-spark-utils/tree/master/nebula-algorithm/src/test/scala/com/vesoft/nebula/algorithm/lib).
+   
+   *Note: The first column of DataFrame in the application are the source vertices, the second are the target vertices and the third are edges' weight.*
```

Comment:
Plural form problem:
The first column of DataFrame in the application represents the source vertices
or
The first column of DataFrame in the application lists the source vertices

So are "the second are xxx" and " the third are".

Reply:
Use the markdown quote for notes would be a better idea.

> **Note:**

Reply:
Done, thanks!